runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/MetricsToCounterUpdateConverter.java

```diff
@@ -111,6 +118,46 @@ public static CounterUpdate fromStringSet(
         .setStringList(stringList);
   }
 
+  public static CounterUpdate fromBoundedTrie(
+      MetricKey key, boolean isCumulative, BoundedTrieData boundedTrieData) {
+    // BoundedTrie uses SET kind metric aggregation which tracks unique strings as a trie.
```

Comment:
This doesn't seem correct.

Reply:
OK. I suppose this value isn't actually used anywhere, and is reasonable enough as informative metadata.

Reply:
I was just looking in that if we can omit it and still cook up something internally to work. 
This is only used on router translation logic when we translate from CounterUpdate to Aggregation to determine combiner type: 

1. https://source.corp.google.com/piper///depot/google3/cloud/dataflow/router/translation/v1beta3.cc;l=1384
2. https://source.corp.google.com/piper///depot/google3/cloud/dataflow/router/translation/v1beta3.cc;l=1083

For BoundedTrie at least for now we don't need kind switch as we have only one kind. 

I do not have a strict preference either ways on this one. Let me know if you prefer one way over the other.

Reply:
Let's leave it as is for now.